docs/src/network-data.md

```diff
@@ -56,7 +78,7 @@ network_data = PowerModels.parse_file("nesta_case3_lmbd.m")
 display(network_data)
 ```
 
-For a detailed list of all possible parameters refer to the specification document provided with [Matpower](http://www.pserc.cornell.edu/matpower/).  
+For a detailed list of all possible parameters refer to the specification document provided with [Matpower](http://www.pserc.cornell.edu/matpower/).
 
 ### Noteworthy Differences from Matpower Data Files
 
```

Comment:
In this list, we should now also mention that loads and shunts are broken out as separate components.